translations/strings.yaml

```diff
@@ -688,7 +688,7 @@ tutorialPopup:
   tutorialLeavePrimaryButtonLabel: Resume tutorial
 
 inAppMessaging:
-  menuTitle: Inbox
+  menuTitle: Messages
```

Comment:
note: this is OK, since this hasn't been translated yet
https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/pull/289/files#diff-23c0d0ff5a7678a6d691121f6e2a4298bd8bbabb5e79f9b5049c14a880cd38e7